temporian/implementation/numpy/data/event.py

```diff
@@ -1,44 +1,37 @@
 from typing import Dict, List, Tuple, Any
 
 import numpy as np
-
+import pandas as pd
 from temporian.core.data.event import Event
 from temporian.core.data.feature import Feature
 from temporian.core.data import dtype
 from temporian.implementation.numpy.data.sampling import NumpySampling
 
 
 class NumpyFeature:
-    def __init__(
-        self, name: str, sampling: NumpySampling, data: np.ndarray
-    ) -> None:
+    def __init__(self, name: str, data: np.ndarray) -> None:
         if len(data.shape) > 1:
             raise ValueError(
                 "NumpyFeatures can only be created from flat arrays. Passed"
                 f" input's shape: {len(data.shape)}"
             )
-        if data.dtype.type not in DTYPE_MAPPING:
-            raise ValueError(
-                f"Unsupported dtype {data.dtype} for NumpyFeature. Supported"
-                f" dtypes: {DTYPE_MAPPING.keys()}"
-            )
+        if data.dtype.type is np.string_:
```

Comment:
What if the dtype is not string and it's also not supported by `DTYPE_MAPPING`?

Reply:
From experimenting, it seems a string can be different "data.dtype" values (e.g. depending on the encoding, or maximum length). "data.dtype.type" is the most consistent solution I found.

Note: Ultimately, we probably want to normalize the exactly used string type e.g. either only allowing one type (with automatic conversion), or be explicit with the exact sub-type.